src/liveEditor/components/FieldToolbar.tsx

```diff
@@ -180,94 +181,125 @@ function FieldToolbarComponent(
                     ]
                 )}
             >
-                {isMultiple ? (
+                {(
+                fieldSchema.data_type == "taxonomy") &&
```

Comment:
Are we sure about this condition? Taxonomy will never be multiple?

Reply:
<img width="1396" alt="Screenshot 2024-09-12 at 11 03 33 AM" src="https://github.com/user-attachments/assets/7c809043-d5aa-43a0-b742-70a55c6de6ce">
You can see in the above image reference